polars/polars-core/src/fmt.rs

```diff
@@ -184,6 +184,13 @@ impl Debug for ListChunked {
     }
 }
 
+// #[cfg(feature = "dtype-fixed-size-list")]
+// impl Debug for FixedSizeListChunked {
+//     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+//         format_array!(f, self, "fixed size list", self.name(), "ChunkedArray")
+//     }
+// }
```

Comment:
If I uncomment this I get the below error. I'm not sure what I'm doing wrong here but presumably I don't want to implement `PolarsNumericType` for `FixedSizeListType`

```rs
error[E0599]: the method `get_any_value` exists for reference `&ChunkedArray<FixedSizeListType>`, but its trait bounds were not satisfied
   --> polars/polars-core/src/fmt.rs:100:32
    |
100 |                     let v = $a.get_any_value(i).unwrap();
    |                                ^^^^^^^^^^^^^ method cannot be called on `&ChunkedArray<FixedSizeListType>` due to unsatisfied trait bounds
...
190 |         format_array!(f, self, "fixed size list", self.name(), "ChunkedArray")
    |         ---------------------------------------------------------------------- in this macro invocation
    |
   ::: polars/polars-core/src/chunked_array/mod.rs:145:1
    |
145 | pub struct ChunkedArray<T: PolarsDataType> {
    | ------------------------------------------ doesn't satisfy `_: ChunkAnyValue`
    |
   ::: polars/polars-core/src/datatypes/mod.rs:55:1
    |
55  | pub struct FixedSizeListType {}
    | ---------------------------- doesn't satisfy `_: PolarsNumericType`
    |
note: trait bound `datatypes::FixedSizeListType: datatypes::PolarsNumericType` was not satisfied
   --> polars/polars-core/src/chunked_array/ops/any_value.rs:180:8
    |
178 | impl<T> ChunkAnyValue for ChunkedArray<T>
    |         -------------     ---------------
179 | where
180 |     T: PolarsNumericType,
    |        ^^^^^^^^^^^^^^^^^ unsatisfied trait bound introduced here
note: the trait `datatypes::PolarsNumericType` must be implemented
   --> polars/polars-core/src/datatypes/mod.rs:242:1
    |
242 | pub trait PolarsNumericType: Send + Sync + PolarsDataType + 'static {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    = note: this error originates in the macro `format_array` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Reply:
> presumably I don't want to implement `PolarsNumericType` for `FixedSizeListType`

I came back to this and realized why I was confused; there's a blanket implementation over everything that implements `PolarsNumericType`, but `ListChunked` had other implementations I needed to copy. So looking at `PolarsNumericType` was a red herring